src/styles/libraries/bootstrap-overrides/_grid.scss

```diff
@@ -16,7 +16,8 @@
 }
 
 .col,
-[class*="col-"] {
+[class^="col-"],
+[class*=" col-"] {
```

Comment:
As we discussed, we will probably want to identify cols at each size at each breakpoint.  Just wanted anybody else seeing this to know what the eventual plan is.

Reply:
For now, we're sticking with this method for ease of implementation since the primary risk is affecting any classes that begin with `col` which is unlikely in the masterclass repo.  The correct implementation will be a bigger and more complex change in the SCSS which will render out each possible column name at each breakpoint and override the gutters.